cernopendata/jsonschemas/records/record-v1.0.0.json

```diff
@@ -523,6 +526,30 @@
     },
     "system_details": {
       "properties": {
+        "container_images": {
+          "items": {
+            "properties": {
+              "description": {
+                "description": "A brief description of the container image",
```

Comment:
It may be useful to specify that the image is used for analysing the data, for example:

- Description of the container image that is recommended for analysing these data

(Alternatively, if we could have several images linked here, one recommended for analysis, one how the data was born, we could even introduce a new property called "purpose" or some such... But at this moment it's probably an overkill, since we plan to store only the images useful for data analyses. Unless there could be several images in the future for different purposes... CC @katilp )

Reply:
Yes, for now, we do not have plans to link to other images than that to be used for analysing the data.

(I could only imagine a separate image for MC to generate a new similar MC dataset with some changes in the input but we are not providing those images and I'd rather not link them to the records in any case)